windows/devices/uwb/UwbDeviceConnector.cxx

```diff
@@ -514,6 +516,8 @@ UwbConnector::HandleNotifications(std::stop_token stopToken)
                     }
                 } else if (lastError == ERROR_MORE_DATA || lastError == ERROR_INSUFFICIENT_BUFFER) {
                     LOG_VERBOSE << "IOCTL_UWB_NOTIFICATION insufficient buffer, " << bytesRequired << " bytes required, current size " << std::size(uwbNotificationDataBuffer) << " bytes";
+                    const UWB_NOTIFICATION_DATA& notificationDataPartial = *reinterpret_cast<UWB_NOTIFICATION_DATA*>(std::data(uwbNotificationDataBuffer));
+                    bytesRequired = notificationDataPartial.size;
                     // Attempt to retry the ioctl with the appropriate buffer size, which is now held in bytesRequired.
                     continue;
                 } else {

```

Comment:
This whole block can probably be removed as long as the driver always handles the request in an asynchronous manner. The simulator driver guarantees this, but I'm unsure of the UwbCx behavior.  Will re-visit this in another PR.